### PR TITLE
Build container image for automation in Kubernetes

### PR DESCRIPTION
**Pull request by troydai**
Let's talk about it tomorrow.

Reference: https://github.com/troydai/a01droid

### CODE REVIEW

**Comment by troydai** [comment]
Once this is in, I will follow up a script to make Travis build docker image for each build.

File: scripts/ci/build.sh [hunk 1]
```diff
@@ -26,9 +26,7 @@ cp $(cd $(dirname $0); pwd)/app/* $app_dir
 
 ##############################################
 # Update version strings
-if $TRAVIS; then
-    $(cd $(dirname $0); pwd)/version.sh $1
-fi
+. $(cd $(dirname $0); pwd)/version.sh $1
```
Comment (johanste) [inline scripts/ci/build.sh]: Nit: I personally prefer to assign more meaningful variable names at the top of my script.

Comment (troydai) [reply scripts/ci/build.sh]: yo. will do in next change.

File: scripts/ci/a01/Dockerfile.py36 [hunk 2]
```diff
@@ -0,0 +1,8 @@
+FROM troydai/azurecli-a01-droid:python3.6-0.8.0
+
+COPY build /tmp/build
+
+RUN find /tmp/build -name '*.whl' | xargs pip install && \
+    find /tmp/build -name '*nspkg*.whl' | xargs pip install
```
[the remark this answers is not in the document]
Comment (troydai) [reply scripts/ci/a01/Dockerfile.py36]: @derekbekoe ?
 
Yeah, I should remove this.

**Comment by troydai** [comment]
@johanste fixed both comments. I will merge in a short period.

**Pull request merged**